superset-frontend/src/explore/actions/exploreActions.ts

```diff
@@ -146,6 +143,26 @@ export function createNewSlice(
 }
 
 export const SLICE_UPDATED = 'SLICE_UPDATED';
-export function sliceUpdated(slice) {
+export function sliceUpdated(slice: Slice) {
   return { type: SLICE_UPDATED, slice };
 }
+
+export const exploreActions = {
+  setDatasourceType,
+  setDatasource,
+  setDatasources,
+  fetchDatasourcesStarted,
+  fetchDatasourcesSucceeded,
+  resetControls,
+  toggleFaveStar,
+  fetchFaveStar,
+  saveFaveStar,
+  setControlValue,
+  setExploreControls,
+  removeControlPanelAlert,
+  updateChartTitle,
+  createNewSlice,
+  sliceUpdated,
+};
+
+export type ExploreActions = typeof exploreActions;
```

Comment:
Explore and enumerate all action functions so it can be used in external typing.